Tasks/GradeAssignment.js

```diff
@@ -0,0 +1,27 @@
+function getGrade(score) {
+  if (score > 100 || score < 0) return "Score should be between 0 and 100.";
+  else {
+    let grade =
```

Comment:
make this logic configurable suppose that the value used in the logic is coming from database how would you implement in that case

Reply:
Please ellaborate this one please

Reply:
there should not be if else statement. we should have to pass data  or globally create in this file. The data should be like [{ grade: A, from: 75, to: 80,}...]

Reply:
if there is no if else, then how would the function decide the grade?

Reply:
this is the part you have to think how would the function decide the grade

Reply:
If you are not able to find solution let me know I will give you a hint.

Reply:
I will find the solution.